docs/core/entity/fan.md

```diff
@@ -103,18 +104,42 @@ ORDERED_NAMED_FAN_SPEEDS = ["one", "two", "three", "four", "five", "six"]  # off
 percentage = ordered_list_item_to_percentage(ORDERED_NAMED_FAN_SPEEDS, "three")
 
 named_speed = percentage_to_ordered_list_item(ORDERED_NAMED_FAN_SPEEDS, 23)
+
+...
+
+    @property
+    def percentage(self) -> Optional[int]:
+        """Return the current speed percentage."""
+        return ordered_list_item_to_percentage(ORDERED_NAMED_FAN_SPEEDS, current_speed)
+
+    @property
+    def speed_count(self) -> Optional[int]:
```

Comment:
This will always return an integer.

Reply:
Fixed in https://github.com/home-assistant/developers.home-assistant/pull/813
Thanks